packages/core/src/extensions/__tests__/configurable-directories.test.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { runInAction } from "mobx";
+import type { ApplicationBuilder } from "../../renderer/components/test-utils/get-application-builder";
+import { getApplicationBuilder } from "../../renderer/components/test-utils/get-application-builder";
+import type { FakeExtensionOptions } from "../../renderer/components/test-utils/get-extension-fake";
+import logErrorInjectable from "../../common/log-error.injectable";
+import getHashInjectable from "../../extensions/extension-loader/file-system-provisioner-store/get-hash.injectable";
+import fsInjectable from "../../common/fs/fs.injectable";
+
+describe("configurable directories for extension files", () => {
+  let builder: ApplicationBuilder;
+  let logErrorMock: jest.Mock;
```

Comment:
I this is not needed?

Reply:
Do you mean `logErrorMock`? That's right, removed.

Reply:
👍